lib/group_domain_models/lib/src/models/write_permissions.dart

```diff
@@ -6,18 +6,15 @@
 //
 // SPDX-License-Identifier: EUPL-1.2
 
-import 'package:sharezone_common/helper_functions.dart';
-
 enum WritePermission { everyone, onlyAdmins }
-WritePermission writePermissionEnumFromString(String data) =>
-    enumFromString(WritePermission.values, data);
-String writePermissionEnumToString(WritePermission writePermissions) =>
-    enumToString(writePermissions);
 
 String writePermissionAsUiString(WritePermission writePermission) {
   switch (writePermission) {
-    case WritePermission.everyone: return 'Alle';
-    case WritePermission.onlyAdmins: return 'Nur Admins';
-    default: return 'Einstellung konnte nicht gefunden werden';
+    case WritePermission.everyone:
+      return 'Alle';
+    case WritePermission.onlyAdmins:
+      return 'Nur Admins';
+    default:
+      return 'Einstellung konnte nicht gefunden werden';
   }
```

Comment:
Couldn't we only use both cases in the switch and put the fallback as a separate return after it so that if a new `WritePermission` case is introduced we get a warning here?